.changeset/few-tigers-grow.md

```diff
@@ -0,0 +1,6 @@
+---
+"@equinor/mad-dfw": major
```

Comment:
When working on packages with version 0.X.X, you can do a `minor` bump instead, once you enter v1, the package is considered `stable`